mavlink-bindgen/src/parser.rs

```diff
@@ -1907,6 +1979,43 @@ pub fn parse_profile(
                             deprecated.as_mut().unwrap().note = Some(t);
                         }
                     }
+                    Some(&MavXmlElement::Param) => {
+                        if let Some(params) = entry.params.as_mut() {
+                            // Some messages can jump between values, like:
+                            // 1, 2, 7
```

Comment:
No need to wrap the comment here (unless rustfmt is being angry about it)

```suggestion
                            // Some messages can jump between values, like: 1, 2, 7
```